src/api/catalog-search/catalog-search.ts

```diff
@@ -155,7 +155,22 @@ export class CatalogSearchAPI {
         return map;
       }, new Map<string, ExtendedCatalogPackage>());
 
-    this.map = this.sort(catalogMap, CatalogSearchSort.PublishDateDesc);
+    this.map = this.sort(
+      catalogMap,
+      // Sort strictly by timestamp
+      (left, right) => {
+        const lDate = new Date(left.metadata.date);
+        const rDate = new Date(right.metadata.date);
+
+        if (lDate === rDate) {
+          return 0;
+        } else if (lDate < rDate) {
+          return 1;
+        } else {
+          return -1;
+        }
+      }
```

Comment:
Why would we not use the same sort function here as we do for results? It would allow us to stick with one tested solution

Reply:
When we show a short-list of "recently updated" packages, I'd wanna look at them strictly by publish date... Otherwise we'll show the same 4 packages for the whole day, and this felt wrong.

Reply:
@RomainMuller I think this is a good argument why we should not compare off date only. Additionally this will lead to inconsistent results for users in different timezones which I would like to avoid